xml/snmp_sysdescr.xml

```diff
@@ -2631,22 +2632,10 @@ Copyright (c) 1995-2005 by Cisco Systems
       <param pos="4" name="os.version.version"/>
    </fingerprint>
 
-   <fingerprint pattern="^(\S+) (.*?) Digital UNIX V(\S+) \(Rev\. ([^\)]+)\).*TCP/IP$">
+   <fingerprint pattern="^(\S+) (.*?) Digital UNIX V(\S+)\s+\(Rev\. ([^\)]+)\).*TCP/IP$">
       <description>Digital/Compaq/HP Tru64 Unix</description>
-      <example>clima.igpsdfpe COMPAQ AlphaServer DS20E 500 MHz Digital UNIX V4.0F (Rev. 1229); Mon Apr 12 13:30:41 EDT 1999 TCP/IP</example>
```

Comment:
A few unique examples are lost, but they don't make a difference to the pattern.